lib/resources/platform.rb

```diff
@@ -0,0 +1,92 @@
+# encoding: utf-8
+
+module Inspec::Resources
+  class PlatformResource < Inspec.resource(1)
+    name 'platform'
+    desc 'Use the platform InSpec resource to test the platform on which the system is running.'
+    example "
+      describe platform.name do
+        it { should eq 'redhat' }
+      end
+
+      describe platform do
+        it { should be_in_family('unix') }
+      end
+    "
+
+    def initialize
+      @platform = inspec.backend.os
```

Comment:
should this be `inspec.backend.platform`?

Reply:
Yeah it should, for now this breaks unit tests as the mock transport in train only mocks up os. I'll put a PR out for train to fix the mock transport and we can update this after.